read.go

```diff
@@ -65,7 +65,15 @@ func (r *reader) readTag() (*tag.Tag, error) {
 func (r *reader) readVR(isImplicit bool, t tag.Tag) (string, error) {
 	if isImplicit {
 		if entry, err := tag.Find(t); err == nil {
-			return entry.VR, nil
+			switch entry.Tag {
+			case tag.PixelData, tag.OverlayData:
+				// OW takes priority in these cases. See notes at:
+				// 1. https://dicom.nema.org/medical/dicom/2024a/output/html/part05.html#sect_8.1.2
+				// 2. https://dicom.nema.org/medical/dicom/2024a/output/html/part05.html#sect_8.2
+				return "OW", nil
+			default:
+				return entry.VRs[0], nil
```

Comment:
out of curiosity, in the case of multiple VRs for a tag in an implicit transfer syntax, does the spec prescribe what to do (e.g. is the first significant here?). This seems fine for now since I'm not sure how else we'd be able to infer the VR more intelligently, even if we tried to inspect the value (may work in some cases).